src/layer/dataset.ts

```diff
@@ -162,6 +162,36 @@ export const DATASET_AWS_LOTL2: Dataset = {
   catalogCollectionId: 'landsat-ot-l2',
 };
 
+export const DATASET_AWS_LTML1: Dataset = {
+  id: 'AWS_LTML1',
+  shJsonGetCapabilitiesDataset: 'LTML1',
+  shWmsEvalsource: 'LTML1',
+  shProcessingApiDatasourceAbbreviation: 'LTML1',
+  datasetParametersType: null,
+  shServiceHostname: 'https://services-uswest2.sentinel-hub.com/',
+  searchIndexUrl: 'https://services-uswest2.sentinel-hub.com/index/v3/collections/LTML1/searchIndex',
+  findDatesUTCUrl: 'https://services-uswest2.sentinel-hub.com/index/v3/collections/LTML1/findAvailableData',
+  orbitTimeMinutes: 99,
+  minDate: new Date(Date.UTC(1982, 7 - 1, 1, 0, 0, 0)), // July 1982
+  maxDate: new Date(Date.UTC(2012, 6 - 1, 1, 0, 0, 0)), // May 2012
```

Comment:
Minor: It says May 2012, but it's actually the 1st of June. I assume this is intentional, as the last date I managed to find for LTML1 is was `2012-05-05`. 

I don't think it's very important for the `maxDate` to be overly precise, so fftr.

Same for LTML2.